wrapper.js

```diff
@@ -26,25 +26,46 @@ function setupMethods (soljson) {
   };
 
   // This calls compile() with args || cb
-  var runWithReadCallback = function (readCallback, compile, args) {
-    // Forward compatibility with 0.6.x
-    if (typeof readCallback === 'object') {
-      readCallback = readCallback.import;
+  var runWithCallbacks = function (callbacks, compile, args) {
+    var readCallback;
+    var smtSolverCallback;
+
+    if (callbacks !== undefined) {
+      readCallback = callbacks.importCallback;
```

Comment:
Actually we called this `import` in 0.5.x.

Reply:
Updated